pkg/util/k8sutil/tpr_util.go

```diff
@@ -128,7 +128,7 @@ func (c *TfJobRestClient) Update(ns string, j *spec.TfJob) (*spec.TfJob, error)
 }
 
 func (c *TfJobRestClient) Delete(ns, name string) error {
-	_, err := c.restcli.Delete().Resource(spec.CRDKindPlural).Namespace(ns).DoRaw()
+	_, err := c.restcli.Delete().Resource(spec.CRDKindPlural).Namespace(ns).Name(name).DoRaw()
```

Comment:
Does this change need to be rebased? I thought you already submitted the fix to the Delete method?

Reply:
Yes, I need to rebase this, duplicated fix.